.github/workflows/ci.yml

```diff
@@ -1,12 +1,9 @@
 name: CI
+
 on:
-  pull_request:
-    branches:
-      - master
   push:
-    branches:
-      - master
-    tags: '*'
+  pull_request:
+
```

Comment:
Actually - should we change tested versions to 1.6, 1, and nightly [here](https://github.com/BioJulia/BioAlignments.jl/pull/77/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR16)